src/components/views/CreateExpense.jsx

```diff
@@ -243,8 +247,11 @@ function CreateExpense(props) {
             ),
           });
         },
-        onError(message, err) {
+        onError(message, err, minimumPayoutUsdValue) {
```

Comment:
Change the name to a more meaningful one, like lessThanMinimumPayout. Notification provider can have access to `minimumPayoutUsdValue` because is defined inside whitelist provider.